crates/sui-core/src/checkpoints/mod.rs

```diff
@@ -803,7 +803,7 @@ impl CheckpointStore {
     // Helper write functions
 
     /// Set the next checkpoint proposal.
-    fn set_proposal(&mut self) -> Result<CheckpointProposal, SuiError> {
+    pub fn set_proposal(&mut self) -> Result<CheckpointProposal, SuiError> {
```

Comment:
Is there a better way to get a proposal out the checkpoint store?

Reply:
This is the way I think.